ssas/service/server.go

```diff
@@ -160,6 +161,7 @@ func (s *Server) LogRoutes() {
 	routes, err := s.ListRoutes()
 	if err != nil {
```

Comment:
We could probably update this section to report an error back if there is one, instead of logging it as informational:
```
	if err != nil {
		ssas.Logger.Errorf("%s routing error: %v", banner, err)
	} 
```

We also don't need to return here, so we could remove that on L164!

Reply:
@laurenkrugen-navapbc would ssas.logger.errorf exit out of the fucntion?

Reply:
nope, it will just [log it at the level `error`](https://pkg.go.dev/github.com/sirupsen/logrus#Errorf)